src/assets/linux/ICONS.md

```diff
@@ -0,0 +1,3 @@
+## Figma
+
+Canonical vector source images for icons can be found on [Figma](https://www.figma.com/file/7V9FN9l9c4zHIls2ZUCoPU/MM-30584-Desktop-App-Icon-Update?node-id=9%3A1170). The files on GitHub are exports.
```

Comment:
This is the README pointing to the master on Figma. It would be better with a deep link, but I don’t know how to do that.